release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -58,10 +58,19 @@ To view bug fixes, see the [GitHub milestone for 2201.0.0 (Swan Lake)](https://g
 
 #### `graphql` package
 
+- Added support for disabling introspection queries
+- Added support for GraphQL interfaces
+- Added support for interfaces implementing interfaces
+- Introduced GraphQL client configurations
+
 ### Improvements
 
 #### `graphql` package
 
+- Added service-level interceptor execution for records fields, maps & tables
+- Added service-level interceptor execution for subscriptions
+- Added returning all the errors related to a GraphQL document in a single response
```

Comment:
```suggestion
- Enabled returning all the errors related to a GraphQL document in a single response
```

Reply:
`Enabled` does not convey the expression here. This is a change to the existing behavior, which returned responses separately for different validation steps. `Enabled` sounds like this is something we have `disabled` previously. Which isn't the case.

Reply:
```suggestion
- Changed to return all the errors related to a GraphQL document in a single response
```

Might be a better option.

Reply:
Shall we say  `Added the ability to return all the errors related to a GraphQL document in a single response`?

Reply:
Still not perfect. This is the behavior now. There's no option to the user to change this behavior.

Reply:
+1 for this: `Changed to return all the errors related to a GraphQL document in a single response`